src/base.lib.php

```diff
@@ -595,13 +666,38 @@ private static function isFirewallAddr()
             return false;
         }
 
-        if (SucuriScanFirewall::getKey()
-            || preg_match('/^192\.88\.13[45]/', $_SERVER['REMOTE_ADDR'])
-            || preg_match('/^185\.93\.(228|229|230|231)/', $_SERVER['REMOTE_ADDR'])
-        ) {
+        if (SucuriScanFirewall::getKey()) {
             return true;
         }
 
+        if (!array_key_exists('HTTP_X_REAL_IP', $_SERVER)) {
```

Comment:
I believe it would be best to use "HTTP_X_SUCURI_CLIENTIP" instead of "HTTP_X_REAL_IP" as X-Real-IP can be sent by other vendors.